### DISCUSSION TITLE
Certain HTML attributes are stripped out, could this behaviour be changed?

### DISCUSSION THREAD

**Discussion by colournetic** [datenstrom/community#817] [Ask a question] [answered]
Hi! On my website, I've used loading=lazy and fetchpriority=low (or high, it depends) on some img tags. However, when inspecting those images in the browser dev tools, it seems the loading and fetchpriority attributes have been stripped out.

Those attributes are not stripped out in files such as navigation.html and footer.html, but they are stripped in page.md files. It looks like Yellow's parser is removing these very useful attributes for improving page speed.

If it is the parser doing this, could this behaviour be changed?

**Comment by markseuffert** [comment, marked as answer]
Yes, I added the `loading` attribute to the HTML whitelist. 

Content files will be filtered trough a HTML whitelist. This means unsafe elements/attributes will be removed automatically. In case you're curious, all safe elements/attributes are defined in the core in function `normaliseData()`. It should be assumed that safe elements/attributes will change over time and we will need to add new ones. You can verify browser support with https://caniuse.com. If you find out that something is missing, please send a pull request or report it as a bug.
